src/main/java/uk/gov/hmcts/reform/idam/api/fr/am/authenticate/AuthenticationChainApi.java

```diff
@@ -0,0 +1,151 @@
+                                   @Param("authIndexValue") String authIndexValue,
+                                   @Param("cookie") String cookie,
+                                   AuthenticationChain requestPayload);
+
+  /**
+   * Submits an authentication chain or blank if initiating the chain
+   * POST authentication chain
+   * Note, this is equivalent to the other <code>authenticate</code> method,
+   * but with the query parameters collected into a single Map parameter. This
+   * is convenient for services with optional query parameters, especially when
+   * used with the {@link AuthenticateQueryParams} class that allows for
+   * building up this map in a fluent style.
+   * @param realm Parameter specifying the realm. (required)
+   * @param cookie  (optional)
+   * @param requestPayload  (optional)
+   * @param queryParams Map of query parameters as name-value pairs
+   *   <p>The following elements may be specified in the query map:</p>
+   *   <ul>
+   *   <li>authIndexType - Specifies the indexType, such as service (required)</li>
+   *   <li>authIndexValue - Specifies the identity of the module we are calling, such as hotpChain (required)</li>
+   *   </ul>
+   * @return AuthenticationChain
+   */
+  @RequestLine("POST /json/realms/{realm}/authenticate?authIndexType={authIndexType}&authIndexValue={authIndexValue}")
+  @Headers({
+  "Content-Type: application/json",
+  "Accept: application/json",
+      "Cookie: {cookie}"
+  })
+  AuthenticationChain authenticate(@Param("realm") String realm,
+                                   @Param("cookie") String cookie,
+                                   AuthenticationChain requestPayload,
+                                   @QueryMap(encoded = true) Map<String, Object> queryParams);
+
+  /**
+   * A convenience class for generating query parameters for the
+   * <code>authenticate</code> method in a fluent style.
+   */
+  public static class AuthenticateQueryParams extends HashMap<String, Object> {
+    public AuthenticateQueryParams authIndexType(final String value) {
+      put("authIndexType", EncodingUtils.encode(value));
+      return this;
+    }
+    public AuthenticateQueryParams authIndexValue(final String value) {
+      put("authIndexValue", EncodingUtils.encode(value));
+      return this;
+    }
+  }
+
+  /**
+   * Submits an authentication chain or blank if initiating the chain
+   * POST authentication chain
+    * @param myRealm Parameter specifying the realm. (required)
+    * @param authIndexType Specifies the indexType, such as service (required)
+    * @param authIndexValue Specifies the identity of the module we are calling, such as hotpChain (required)
+    * @param cookie  (optional)
+    * @param requestPayload  (optional)
+   * @return AuthenticationChain
+   */
+  @RequestLine("POST /json/realms/{myRealm}/authenticate?authIndexType={authIndexType}&authIndexValue={authIndexValue}")
```

Comment:
did you try just not having the body defined in this method?

Reply:
pushing a change for this, tested and it works 👍